src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1101,11 +1101,13 @@ private ProcessStartInfo GetProcessStartInfo(bool redirectOutput, bool redirectE
                 if (redirectOutput)
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-windows`
TestAppDomainProcessExitEvenHandlerNotLeaking
```powershell
Assert.DoesNotContain() Failure
Found:    (filter expression)
In value: Delegate[] [EventHandler { Method = Void DisposeOnShutdown(System.Object, System.EventArgs), Target = null }, EventHandler { Method = Void CurrentDomain_ProcessExit(System.Object, System.EventArgs), Target = null }]
   at PSTests.Sequential.RunspaceTests.TestAppDomainProcessExitEvenHandlerNotLeaking() in D:\a\1\s\test\xUnit\csharp\test_Runspace.cs:line 127
```
Resources strings in System.Management.Automation (was -ResGen used with Start-PSBuild).'ParserStrings' should be an available type and the strings should be correct
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ResourceValidation\TestRunner.ps1: line 63
63:                     $resourceStringToCheck = $resourceType.GetProperty($inResource.name,$bindingFlags).GetValue(0)

```